src/main/webapp/app/variables.scss

```diff
@@ -16,6 +16,11 @@ $danger: #dc3545;
 $link-hover-color: $oncokb-darker-blue;
 $nav-bg-color: $oncokb-blue;
 
+$breadcrumb-bg: transparent;
+$breadcrumb-padding-x: 0;
+$breadcrumb-padding-y: 0;
```

Comment:
Padding of `0` is default if you don't define them here?

Reply:
I can double check, I don't think they are 0.

Reply:
This is the default.
```
$breadcrumb-padding-y:              .75rem !default;
$breadcrumb-padding-x:              1rem !default;
```